bratshelper/tests.go

```diff
@@ -193,7 +193,7 @@ func StagingWithADepThatIsNotTheLatest(depName string, copyBrats func(string) *c
 	StagingWithADepThatIsNotTheLatestConstrained(depName, "x", copyBrats)
 }
 
-func StagingWithCustomBuildpackWithCredentialsInDependencies(depRegexp string, copyBrats func(string) *cutlass.App) {
+func StagingWithCustomBuildpackWithCredentialsInDependencies(copyBrats func(string) *cutlass.App) {
```

Comment:
This is a breaking change. Will need to change all buildpacks calling this method (to the simpler though).